src/containers/NotificationToggle.js

```diff
@@ -0,0 +1,58 @@
+import React, { useEffect, useState } from 'react';
+import styled from 'styled-components';
+import { H2 } from '../components/Typography';
+import ToggleSwitch from '../components/ToggleSwitch';
+import notifications from '../utility/notifications';
+import {
+  NOTIFICATION_PERMISSIONS as N_PERMISSIONS,
+  NOTIFICATION_SETTINGS_CACHE_KEY as N_SETTINGS_CACHE_KEY
+} from '../utility/Constants';
+
+const NotificationToggleContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+`
+
+const StyledH2 = styled(H2)`
+  margin: 0 0 0 0.5em;
+  opacity: 1;
+`
+
+export default () => {
+  const [toggled, setToggled] = useState(false)
+
+  useEffect(() => {
+    setToggled(notifications.areEnabled())
+  }, [])
+
+  const handleToggle = (e) => {
+    if (notifications.isCurrentPermission(N_PERMISSIONS.DEFAULT)) {
+      notifications.requestPermission(permission => {
+        toggleNotifications(permission === N_PERMISSIONS.GRANTED)
+      })
+    }
```

Comment:
is this asking permission to enable notifications for the first time? might be good to add a comment here